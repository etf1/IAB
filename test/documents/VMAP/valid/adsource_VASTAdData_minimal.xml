<!-- An AdSource with minimal VASTAdData -->
<vmap:VMAP version='1.0' xmlns:vmap='http://www.iab.net/vmap-1.0'>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start" >
        <vmap:AdSource allowMultipleAds='false' followRedirects='true' id='1' >
            <vmap:VASTAdData>
                <VAST version='3.0'></VAST>
            </vmap:VASTAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
</vmap:VMAP>
