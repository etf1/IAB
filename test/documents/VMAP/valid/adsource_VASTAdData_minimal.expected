{
  "version": "1.0",
  "breaks": [
    {
      "timeOffset": "start",
      "breakTypes": [
        1
      ],
      "source": {
        "id": "1",
        "allowMultipleAds": false,
        "followRedirects": true,
        "dataType": 1,
        "VASTAdData": {
          "version": "3.0",
          "ads": []
        }
      },
      "id": "cue_point-347762684"
    }
  ]
}
